writer/table.go

```diff
@@ -39,12 +41,43 @@ func (t TableWriter) Write(writer io.Writer) error {
 
 	table.Render()
 
+	// Disable the Output Summary if there are no outputs to display
+	if t.outputs == true {
```

Comment:
Instead of adding new outputs flag, `if len(t.output_changes) > 0` shall we display outputs table ?